### PR TITLE
Add support for php8.1

### PR DESCRIPTION
**Pull request by satrun77**

### CODE REVIEW

**Comment by satrun77** [comment]
👆 @Jianbinzhu 👈

File: src/models/SiteBanner.php [hunk 1]
```diff
@@ -191,8 +191,8 @@ public function isActive()
             return true;
         }
 
-        $startDate = new DateTime($this->StartDate);
-        $endDate   = new DateTime($this->EndDate);
+        $startDate = new DateTime($this->StartDate ?? 'now');
```
Comment (Jianbinzhu) [inline src/models/SiteBanner.php]: `new DateTime('now')` could be different to 
`new DateTime(DBDatetime::now()->Format(DBDatetime::ISO_DATETIME))`?

Comment (satrun77) [reply src/models/SiteBanner.php]: Updated! Thanks

**Review by Jianbinzhu — APPROVED** [review]
🚀

**Pull request merged**